portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,100 @@
   width: 650px;
 }
 
+h1 {
+	font-size: 50px;
+}
+
+h2 {
+	font-size: 36px;
+}
+
+p {
+  font-size: 25px;
+}
+
 #greeting-container {
   margin-top: 20px;
 }
+
+#footer {
+   bottom : 2px;  
+   height : 40px;  
+   margin-top : 40px;  
+   text-align: center;  
+   vertical-align: middle; 
+   position:fixed;
+   width:100%;
+}
+
+#projects {
+  margin-left: auto;
+  margin-right: auto;
+  font-family: 'Caveat';
+
```

Comment:
nit: please remove this blank line

(another note about my review style...I'll typically prefix super minor, nit-picky comments with "nit: ")